crates/core/tedge_agent/src/agent.rs

```diff
@@ -147,13 +151,17 @@ impl SmAgentConfig {
         let tedge_log_dir = PathBuf::from(&format!("{tedge_log_dir}/{AGENT_LOG_PATH}"));
         let tedge_run_dir = tedge_config.query_string(RunPathSetting)?.into();
 
+        let bind_address = tedge_config.query(MqttBindAddressSetting)?;
```

Comment:
We might have to add an `HttpPortSetting` as well in future.

Reply:
This might even be required in a near future ... as highlighted by the following comment on the port number.